packages/api/test/test-suites/filters.ts

```diff
@@ -146,12 +162,6 @@ export function runFilterTests(
       });
     });
 
-    it("should return the correct results when filtering by a 'null' rollup", async () => {
```

Comment:
@xFJA this test still makes sense, right?

Reply:
I think it's a merge-related error